### PR TITLE
fix Cyrillic letters turning into ☐☐☐

### PR DESCRIPTION
**Pull request by DetectiveEnotsky**
### Purpose
 Basically, most of the Cyrillic symbols are turning into ☐☐☐ when typing them into chat. This should fix it, by adding a fallback font asset with cyrillic symbols. Partially fixеs #6774

### Notes:
 ATTENTION. This code may not work. I haven't tested it, yet(and i'm not going to), because my PC explodes every time I press Build and Run in my UnityEditor. 
UPD: I tested it. Worked as intended.  <br>
image

### Changelog:

CL: [Fix] Fixes Cyrillic letterns turning into  ☐

### CODE REVIEW

File: UnityProject/Packages/packages-lock.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
       "dependencies": {}
     },
     "com.unity.2d.tilemap.extras": {
-      "version": "2.2.4",
+      "version": "2.2.5",
```
Comment (corp-0) [inline UnityProject/Packages/packages-lock.json]: I think you shouldn't push this file

Comment (DetectiveEnotsky) [reply UnityProject/Packages/packages-lock.json]: Yeah, probably. I'll try and do it now.

Comment (DetectiveEnotsky) [reply UnityProject/Packages/packages-lock.json]: Okie dokie, fixed in the commit 7362e4426c635b30629ec37d210a48a3f654fb37...

**Comment by DetectiveEnotsky** [comment]
Okie dokie, now it's ready. I think.

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**